Xamarin.Essentials/AppInfo/AppInfo.netstandard.cs

```diff
@@ -11,5 +11,7 @@ public static partial class AppInfo
         static string PlatformGetBuild() => throw ExceptionUtils.NotSupportedOrImplementedException;
 
         static void PlatformShowSettingsUI() => throw ExceptionUtils.NotSupportedOrImplementedException;
+
+        static WindowSize PlatformWindowAppSize() => throw ExceptionUtils.NotSupportedOrImplementedException;
```

Comment:
I am curious if we should be returning an IEnumerable of this... Apps can now have multiple windows....

Reply:
For start we can do "

static AppViewInfo CurrentView {get;}

And then in the future we could return IEnumerable based on demand. This is how we structured the displays